include/dev/simplebus.h

```diff
@@ -0,0 +1,24 @@
+#ifndef _DEV_SIMPLEBUS_H_
+#define _DEV_SIMPLEBUS_H_
+
+typedef uint32_t phandle_t;
+typedef struct device device_t;
```

Comment:
Let's include real source of these typedefs.

Reply:
Done.